modules/nf-core/cnvkit/export/main.nf

```diff
@@ -0,0 +1,35 @@
+process CNVKIT_EXPORT {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::cnvkit=0.9.9 bioconda::samtools=1.16.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/cnvkit:0.9.9--pyhdfd78af_0':
+        'quay.io/biocontainers/cnvkit:0.9.9--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta) , path(cns)
+
+    output:
+    tuple val(meta), path("${prefix}.${suffix}"), emit: output
+    path "versions.yml"                         , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    prefix = task.ext.prefix ?: "${meta.id}"
+    suffix = task.ext.suffix ?: "bed"
```

Comment:
we can use args to assess the suffix directly.
No need to add a new `ext`

Reply:
cf https://github.com/nf-core/modules/blob/2f2a92e046d593bd50ac89715d81e10d9ad67ade/modules/nf-core/ensemblvep/main.nf#L31